api-contract/src/test/java/uk/gov/hmcts/payment/api/contract/FeeDtoTest.java

```diff
@@ -0,0 +1,62 @@
+package uk.gov.hmcts.payment.api.contract;
+
+import java.util.Iterator;
+import java.util.Set;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import javax.validation.constraints.Positive;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertTrue;
+
+import org.junit.Before;
+import org.junit.Test;
+
+public class FeeDtoTest {
+    private static Validator validator;
+
+    @Before
+    public void setUp() {
+        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+        validator = factory.getValidator();
+    }
+
+    @Test
+    public void negativeFeeVolumeShouldFailValidation() {
+        FeeDto feeDto = new FeeDto();
+        feeDto.setVolume(-1);
+
+        Set<ConstraintViolation<FeeDto>> violations = validator.validate(feeDto);
+        Iterator<ConstraintViolation<FeeDto>> iterator = violations.iterator();
+        while (iterator.hasNext()) {
```

Comment:
May be AssertJ simplifies like this 
assertThat(iterator).extracting(ConstraintViolation::getMessage)
            .contains("must be greater than 0");